driver_test.go

```diff
@@ -546,6 +547,61 @@ func testInt(t *testing.T, arrow bool) {
 	})
 }
 
+func TestFixedArrow(t *testing.T) {
+	runTests(t, dsn, func(dbt *DBTest) {
+
+		dbt.mustExec("ALTER SESSION set go_query_result_format = arrow_force")
+		//rows := dbt.mustQuery("SELECT 10.0::NUMBER(38,36)")
+		//rows := dbt.mustQuery("SELECT 23456.1::NUMBER(31,20)")
```

Comment:
can you remove these two lines if not required?

Reply:
ok, let's add `23456.1::NUMBER(31,20)` test case to receive decimal128 for now and focus other data type conversions.